lib/code_corps/policy/policy.ex

```diff
@@ -27,6 +27,9 @@ defmodule CodeCorps.Policy do
   defp can?(%User{} = user, :create, %Comment{}, %{} = params), do: Policy.Comment.create?(user, params)
   defp can?(%User{} = user, :update, %Comment{} = comment, %{}), do: Policy.Comment.update?(user, comment)
 
+  defp can?(%User{} = user, :create, %Organization{}, _), do: Policy.Organization.create?(user)
```

Comment:
We prefer keeping functions with identical arity grouped up. Credo gives out a warning when that's not the case to, so I'd remove this blank line. Also, I know the last argument doesn't matter in this case, but the spec at the top says it's a map, so `%{}` instead of `_` is still prefered.